src/stories/styles/select.css

```diff
@@ -0,0 +1,13 @@
+.flex {
```

Comment:
bu css global scope a gidebilir bunu silip inline storyde inline yapalim veya scss acip basina bir wrapper koyalim

Reply:
bunu, stories için yazdım. Önemsiz bir css dosyası. Bence gerek yok?